Assets/VRTK/SDK/Simulator/SDK_ControllerSim.cs

```diff
@@ -0,0 +1,69 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+
+public class SDK_ControllerSim : MonoBehaviour 
+{
+    private Vector3 lastPos;
+    private Vector3 lastRot;
+    private List<Vector3> posList;
+    private List<Vector3> rotList;
+    private bool selected;
+
+    public bool Selected
+    {
+        get
+        {
+            return selected;
+        }
+        set
+        {
+            selected = value;
+        }
+    }
+
+    private void Awake()
```

Comment:
private methods should come after public and protected, so just move Awake and Update to the bottom of the file.